lib/resty/auto-ssl/storage.lua

```diff
@@ -65,6 +65,10 @@ function _M.set_cert(self, domain, fullchain_pem, privkey_pem, cert_pem, expiry)
   return self.adapter:set(domain .. ":latest", string)
 end
 
+function _M.delete_cert(self, domain)
+  return self.adapter:delete(domain .. ":latest")
```

Comment:
Assuming we keep the ability to retain some old certificates (as discussed in https://github.com/GUI/lua-resty-auto-ssl/pull/124), should this `delete_cert` also delete all of the timestamped older copies for the domain?

While deleting all the versions might somewhat defeat the purpose of retaining those backups, I'm not sure we'd have a very clean way to remove the old timestamped versions once the `latest` version is gone (since it won't be part of the renewal process), so I'm maybe inclined to delete all of them if this gets triggered. But I could really go either way on this.